--- origsrc/roadnav-0.19/Makefile.am	2007-10-08 21:02:14.000000000 -0500
+++ src/roadnav-0.19/Makefile.am	2008-07-01 00:59:53.640625000 -0500
@@ -1,7 +1,6 @@
 SUBDIRS = src data win32 skins scripts debian
 
-roadnavdocdir = ${prefix}/share/doc/roadnav
-roadnavdoc_DATA = \
+noinst_DATA = \
 	README\
 	COPYING\
 	AUTHORS\
@@ -10,7 +9,7 @@
 	NEWS\
 	TODO
 
-EXTRA_DIST = $(roadnavdoc_DATA) autogen.sh roadnav.spec roadnav.desktop
+EXTRA_DIST = $(noinst_DATA) autogen.sh roadnav.spec roadnav.desktop
 
 dist-hook:
 	find $(distdir) -exec touch -r Makefile \{\} \;
--- origsrc/roadnav-0.19/src/CompassControl.h	2007-10-08 21:02:14.000000000 -0500
+++ src/roadnav-0.19/src/CompassControl.h	2008-07-01 00:31:49.328125000 -0500
@@ -30,8 +30,9 @@
 #define COMPASSCONTROL_H
 
 #include <wx/wx.h>
+#include <libroadnav/MapControl.h>
 
-DECLARE_EVENT_TYPE(wxEVT_SAFE_REFRESH, -1)
+DECLARE_LOCAL_EVENT_TYPE(wxEVT_SAFE_REFRESH, -1)
 
 //////////////////////////////////////////////////////////////////////////////
 ///
--- origsrc/roadnav-0.19/src/Makefile.am	2007-10-08 21:02:14.000000000 -0500
+++ src/roadnav-0.19/src/Makefile.am	2008-07-01 00:03:15.687500000 -0500
@@ -176,4 +176,4 @@
 	MainPage.txt
 
 install-data-hook:
-	- test -w $(DESTDIR)/usr/share/pixmaps && $(INSTALL_DATA) appicon48.png $(DESTDIR)/usr/share/pixmaps/roadnav.png
+	- test -w $(DESTDIR)/usr/share/pixmaps && $(INSTALL_DATA) $(srcdir)/appicon48.png $(DESTDIR)/usr/share/pixmaps/roadnav.png
--- origsrc/roadnav-0.19/src/SatelliteControl.h	2007-10-08 21:02:14.000000000 -0500
+++ src/roadnav-0.19/src/SatelliteControl.h	2008-07-01 00:22:03.953125000 -0500
@@ -35,7 +35,6 @@
 #include "CompassControl.h"
 #include "GPSMonitorThread.h"
 
-DECLARE_EVENT_TYPE(wxEVT_SAFE_REFRESH, -1)
 
 //////////////////////////////////////////////////////////////////////////////
 ///
--- origsrc/roadnav-0.19/src/SolarTimer.h	2007-10-08 21:02:14.000000000 -0500
+++ src/roadnav-0.19/src/SolarTimer.h	2008-07-01 00:12:39.328125000 -0500
@@ -39,7 +39,7 @@
 ///
 //////////////////////////////////////////////////////////////////////////////
 BEGIN_DECLARE_EVENT_TYPES()
-    DECLARE_EVENT_TYPE(wxEVT_SOLAR, wxEVT_USER_FIRST)
+    DECLARE_LOCAL_EVENT_TYPE(wxEVT_SOLAR, wxEVT_USER_FIRST)
     DECLARE_EVENT_TYPE(wxEVT_TIMECHANGE, wxEVT_USER_FIRST+1)
 END_DECLARE_EVENT_TYPES()
 
--- origsrc/roadnav-0.19/src/wxGPSEvent.h	2007-10-08 21:02:14.000000000 -0500
+++ src/roadnav-0.19/src/wxGPSEvent.h	2008-07-01 00:13:42.390625000 -0500
@@ -73,7 +73,7 @@
 		wxString m_strErrorMessage;
 };
 
-DECLARE_EVENT_TYPE(wxEVT_GPS_ACTION, -1)
+DECLARE_LOCAL_EVENT_TYPE(wxEVT_GPS_ACTION, -1)
 
 typedef void (wxEvtHandler::*wxGPSEventFunction)(wxGPSEvent&);
 
